test/starlark_tests/apple_static_xcframework_import_tests.bzl

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""apple_static_xcframework_import Starlark tests."""
+
+load(":rules/analysis_failure_message_test.bzl", "analysis_failure_message_test")
+load(":rules/common_verification_tests.bzl", "archive_contents_test")
+
+def apple_static_xcframework_import_test_suite(name):
+    """Test suite for apple_static_xcframework_import.
+
+    Args:
+      name: the base name to be used in things created by this macro
+    """
+
+    # # Verify importing XCFramework with static frameworks (i.e. not libraries) fails.
```

Comment:
Same, a comment here?

Reply:
i will toss this one as well, i just want to make sure we have sufficient coverage in this path, which i think we do from our existing rule, but i need to verify